autogen/main/cluster.tf.tmpl

```diff
@@ -610,9 +607,10 @@ resource "google_container_node_pool" "pools" {
       for_each = local.cluster_node_metadata_config
 
       content {
-        node_metadata = lookup(each.value, "node_metadata", workload_metadata_config.value.node_metadata)
+        mode = lookup(each.value, "node_metadata", workload_metadata_config.value.mode)
```

Comment:
I'm not sure we want to change the *input* value (ie. still look at `node_metadata`).

Reply:
I need to refresh my memory on this (and find a line reference), but I think I'm still using the original _input_value, but I've adjusted the `workload_metadata_config` object to match the names of the new properties, so it serves as an adapter between the two; at the time that seemed to make the most sense to me.